test/conditionals_test.dart

```diff
@@ -13,6 +13,35 @@ import 'package:rohd/src/exceptions/sim_compare/sim_compare_exceptions.dart';
 import 'package:rohd/src/utilities/simcompare.dart';
 import 'package:test/test.dart';
 
+import 'counter_wintf_test.dart';
+
+class ResetFlipflops extends Module {
```

Comment:
Why not add this test module and test to the `coutner_wintf_test.dart` file instead, so both implementations of the same module are in the same place?  You could maybe even parameterize the same `Counter` module to conditionally generate the reset two different ways, then reuse the same set of vectors.  There's too much copy-paste here, I think.